graphql/util.go

```diff
@@ -419,6 +419,10 @@ func SchedulePatch(ctx context.Context, patchId string, version *model.Version,
 			if err = p.SetChildPatches(); err != nil {
 				return errors.Wrapf(err, "error attaching child patches '%s'", p.Id.Hex()), http.StatusInternalServerError, "", ""
 			}
+			p.Triggers.Aliases = patchUpdateReq.PatchTriggerAliases
```

Comment:
I thought we did this in a previous pr. did we do something similar somewhere else? It's not important, I'm just wondering....

Reply:
Almost! We set `p.Triggers.ChildPatches` [here](https://github.com/evergreen-ci/evergreen/blob/main/units/patch_intent.go#L525), added in https://github.com/evergreen-ci/evergreen/pull/4947. So this will ensure the two fields are properly zipped!

Reply:
ah yes!